### PR TITLE
Simplify getting IP address

### PR DESCRIPTION
**Pull request by friism**
# Simplify getting IP address
## Summary

Doc update to avoid `docker exec` and user parsing `ipconfig` output

### CODE REVIEW

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/framework/docker/aspnetmvc?branch=pr-en-us-1107](https://review.docs.microsoft.com/en-us/dotnet/articles/framework/docker/aspnetmvc?branch=pr-en-us-1107)  

**Status**: SucceededWithWarning. **Message**: Warning occurred: Unable to resolve cross-reference "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptKeyHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptKeyHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptProviderHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptProviderHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptSecretHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptSecretHandle</a>"..  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C10%5C3%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201610032005052449-1107%5Cworkflow_report.html).

**Comment by mairaw** [comment]
@BillWagner @spboyer can you please review this one?

File: docs/framework/docker/aspnetmvc.md [hunk 1]
```diff
@@ -186,26 +186,11 @@ The `mvcrandomanswers` is the name of the image to start.
...
-
-
-Ethernet adapter Ethernet 2:
-
-   Connection-specific DNS Suffix  . :
-   Link-local IPv6 Address . . . . . : fe80::91d:ce97:dd27:460d%5
-   IPv4 Address. . . . . . . . . . . : 172.31.194.61
-   Subnet Mask . . . . . . . . . . . : 255.240.0.0
-   Default Gateway . . . . . . . . . : 172.16.0.1
+docker inspect -f "{{ .NetworkSettings.Networks.nat.IPAddress }}" randomanswers
```
Comment (spboyer) [inline docs/framework/docker/aspnetmvc.md]: While this does in fact get straight to the `ipaddress`, Windows users/admins are accustomed to using the `ipconfig` command and seeing the output. Can we combine the original content and add this line as a "shortcut" stating as such?

**Review by spboyer — COMMENTED** [review]
see inline comment

**Pull request merged**